opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -279,36 +279,42 @@ namespace Opm
 
             // compute volume ratio between connection at standard conditions
             Value volumeRatio = bhp * 0.0; // initialize it with the correct type
-;
+
             if (FluidSystem::enableVaporizedWater() && FluidSystem::enableDissolvedGasInWater()) {
                 disOilVapWatVolumeRatio(volumeRatio, rvw, rsw, pressure,
                                         cmix_s, b_perfcells_dense, deferred_logger);
+                // only supported for gas-water case
```

Comment:
I leave @bska and @akva2  to comment whether we want to use `assert` here.  

It basically means that we enter a simulation not allowed at the moment, which is have water-gas mixing while we might not have water or gas, or we have oil active.  We do not support that.

Reply:
even though we ideally would have failed earlier, using assert for things that can be triggered by user input (files) is never the correct solution.

Reply:
The Simulator will throw in the setup face. I added the asserts for the programmer if/when we start thinking about adding disgasw for a blackoil run. I can remove them and add a throw instead.

Reply:
if it throws earlier it's all good.